sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Tests.csproj

```diff
@@ -29,7 +29,7 @@
 		
 		<PackageReference Include="nunit" />
 		<PackageReference Include="Moq" />
-		<PackageReference Include="NUnit3TestAdapter"/>
+		<PackageReference Include="NUnit3TestAdapter" />
 		
 		<PackageReference Include="xunit.runner.visualstudio">
```

Comment:
The xUnit runner package can be removed now that you've converted to NUnit.

Reply:
removed